cmd/skopeo/list_tags.go

```diff
@@ -113,13 +117,20 @@ func (opts *tagsOptions) run(args []string, stdout io.Writer) (retErr error) {
 	}
 
 	// Do transport-specific parsing and validation to get an image reference
-	imgRef, err := parseDockerRepositoryReference(args[0])
-	if err != nil {
+	var imgRef types.ImageReference
+	if err = retryIfNecessary(ctx, func() error {
+		imgRef, err = parseDockerRepositoryReference(args[0])
+		return err
+	}, opts.retryOpts); err != nil {
```

Comment:
This is not necessary; `parseDockerRepositoryReference` is happens purely in local memory.